### PR TITLE
add models and constants for RML

### PR DESCRIPTION
**Pull request by diitaz93**
## Description
Closes: https://github.com/Clinical-Genomics/improve-order-flow/issues/71
Adds models and constants for RML order validation

### Added

- Rml Sample model
- Rml Order model
- RMLDeliveryData class

### Changed

-

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Review by ChrOertlin — APPROVED** [review]
lgtm!

File: cg/services/order_validation_service/workflows/rml/models/sample.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from cg.models.orders.sample_base import ControlEnum, PriorityEnum
+from cg.services.order_validation_service.models.sample import Sample
+
+
+class RmlSample(Sample):
+    control: ControlEnum | None = None
+    index: str
+    index_number: int
+    pool: str | None = None
+    pool_concentration: float | None = None
```
Comment (islean) [inline cg/services/order_validation_service/workflows/rml/models/sample.py:10]: I think these are non-optional

File: cg/services/order_validation_service/workflows/rml/models/sample.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from cg.models.orders.sample_base import ControlEnum, PriorityEnum
+from cg.services.order_validation_service.models.sample import Sample
+
+
+class RmlSample(Sample):
```
Comment (islean) [inline cg/services/order_validation_service/workflows/rml/models/sample.py:5]: Minor thing - for RML the volume is a mandatory field. We should consider overriding the volume attribute set on the parent class to mark it as non-optional here

**Review by islean — APPROVED** [review]
Looks good!

**Pull request merged**